node/core/dispute-coordinator/Cargo.toml

```diff
@@ -11,6 +11,8 @@ parity-scale-codec = "3.1.2"
 kvdb = "0.11.0"
 thiserror = "1.0.30"
 lru = "0.7.5"
+lru-cache = "0.1.2"
```

Comment:
why are we using two different crates for the same thing?
lru-cache is also in maintenance mode

Reply:
That's a good point, I switched to this while I was trying to refactor using references, but I don't recall the exact reason 🥲 . I'll drop it.